### PR TITLE
Update introduction.md for `blaze` project

### PR DESCRIPTION
**Pull request by liyuance**
Add the Blaze project to known users.

## Which issue does this PR close?

Closes #.

### CODE REVIEW

File: docs/source/user-guide/introduction.md [hunk 1]
```diff
@@ -96,6 +96,7 @@ Here are some active projects using DataFusion:
 
 - [Arroyo](https://github.com/ArroyoSystems/arroyo) Distributed stream processing engine in Rust
 - [Ballista](https://github.com/apache/datafusion-ballista) Distributed SQL Query Engine
+- [Blaze](https://github.com/kwai/blaze) The Blaze accelerator for Apache Spark leverages native vectorized execution to accelerate query processing
```
Comment (alamb) [inline docs/source/user-guide/introduction.md:99]: There is an existing entry for `blaze` on line 128 below this. The repo is different though. 

Is this the same project or a different one?

If this is the same blaze project, can you please remove the duplicate listing below?

**Review by alamb — COMMENTED** [review]
Thank you @liyuance  -- I am glad to see blaze on this list 🙏

File: docs/source/user-guide/introduction.md (hunk 1 above)
Comment (liyuance) [reply docs/source/user-guide/introduction.md:99]: Thanks for your reminding, I have removed 'blaze' from  less active project list.

**Review by alamb — APPROVED** [review]
Thank you @liyuance  -- looks great!

**Comment by alamb** [comment]
Thanks again!

**Pull request merged**